src/pages/error/404.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
filenamenya langsung `index.tsx` aja dhan. kurang enak dilihat kalo filename full angka semua. hehe. thank you